lib/scripts/test-external.js

```diff
@@ -0,0 +1,118 @@
+'use strict';
+/* eslint-disable no-console, node/no-extraneous-require, node/no-unpublished-require */
+const fs = require('fs');
+const path = require('path');
+const { execSync } = require('child_process');
```

Comment:
lets use `execa` as it cleans up leaked children, if the script is exited early.

Reply:
@stefanpenner this? https://github.com/sindresorhus/execa